Controllers/AdminController.cs

```diff
@@ -33,9 +37,14 @@ public IActionResult Index(long id)
         {
             //TODO: Index will return the list of books (with Author), ordered by SKU
             //or a single book (with Author), if passed an id > 0
-            IEnumerable<Book> books = _db.Books.Include(b => b.Author);
-
+            IEnumerable<Book> books = _db.Books.Include(b => b.Author)
+            .OrderBy(b => b.SKU);
 
+            if (id > 0)
+            {
+                books = books.Where(b => b.Id == id);
```

Comment:
Good work. You also need to use `.Include(b => b.Author)` in line 45, so that the book has the Author's name